revolv/templates/project/project.html

```diff
@@ -13,223 +14,344 @@
+                                                            <span class="our_apps">Our Appritiation</span>
+                                                            <p class="dummy">Lorem ipsum Proim gravida nibh vel  nibh vel velit.</p>
+                                                            <a class="btn-blue btn-i-want-to-donate" id="donate-button" href="/signin/?next=/project/1/&amp;reason=donate#login">
+                                                                <p>DONATE</p>
+                                                            </a>
+                                                        </div>
+                                                        <!-- end .module-box -->
+                                                    </aside>
+                                                </div>
+                                            </div>
+                                            <div class="col-lg-12 col-md-12 col-sm-4 col-xs-4">
+                                                <div class="right_asides">
+                                                    <aside class="right-aside">
+                                                        <div class="module-box payment-module">
+                                                            <h1 class="basic">Silver</h1>
+                                                            <strong class="dollar">$ 40</strong>
+                                                            <span class="our_apps">Our Appritiation</span>
+                                                            <p class="dummy">Lorem ipsum Proim gravida nibh vel velit auctor aliquet vel velit auctor aliquet</p>
+                                                            <a class="btn-blue btn-i-want-to-donate" id="donate-button" href="/signin/?next=/project/1/&amp;reason=donate#login">
+                                                                <p>DONATE</p>
+                                                            </a>
+                                                        </div>
+                                                        <!-- end .module-box -->
+                                                    </aside>
+                                                </div>
+                                            </div>
+                                            <div class="col-lg-12 col-md-12 col-sm-4 col-xs-4 pull-right">
+                                              <div class="right_asides">
+                                                <aside class="right-aside">
+                                                    <div class="module-box payment-module">
+                                                        <h1 class="basic">Gold</h1>
+                                                        <strong class="dollar">$ 80</strong>
+                                                        <span class="our_apps">Our Appritiation</span>
```

Comment:
Please use dashes instead of underscores for class names. For example: `our-apps` instead of `our_apps`